### PR TITLE
Greenwave consumer : use the message data for single build updates

### PR DESCRIPTION
**Pull request by abompard**
See #3368.

### CODE REVIEW

File: bodhi/server/consumers/greenwave.py [hunk 1]
```diff
@@ -68,5 +68,28 @@ def __call__(self, message: fedora_messaging.api.Message):
...
+        Extract gating information from the Greenwave message and return it.
+
+        Returns:
+            TestGatingStatus:
+                - TestGatingStatus.ignored if no tests are required
+                - TestGatingStatus.failed if policies are not satisfied
+                - TestGatingStatus.passed if policies are satisfied, and there
+                  are required tests
+        """
+        if not msg['policies_satisfied']:
```
Comment (pypingou) [inline bodhi/server/consumers/greenwave.py:95]: just to be on the safe side: ``msg.get("policies_satisfied")`` here?

Comment (pypingou) [reply bodhi/server/consumers/greenwave.py:95]: or better, we check if that key is in the dict first and bail if it isn't then move on to your code as is :)

Comment (abompard) [reply bodhi/server/consumers/greenwave.py:95]: Sure, why not, I added that.

**Comment by abompard** [comment]
@pypingou , are you happy with the changes?

File: bodhi/server/consumers/greenwave.py [hunk 2]
```diff
@@ -50,6 +50,7 @@ def __call__(self, message: fedora_messaging.api.Message):
             log.debug("Ignoring message without body.")
             return
 
+        print(msg)
```
Comment (pypingou) [inline bodhi/server/consumers/greenwave.py]: debugging left over?

**Comment by pypingou** [comment]
one comment the rest looks fine :)

File: bodhi/server/consumers/greenwave.py (hunk 2 above)
Comment (abompard) [reply bodhi/server/consumers/greenwave.py]: Indeed! It should be OK now, sorry.

**Review by pypingou — APPROVED** [review]
Thanks!

**Pull request merged**